src/lib/y2network/connection/connection.rb

```diff
@@ -0,0 +1,20 @@
+require "y2network/interface"
+
+module Y2Network
+  module Connection
+    class Connection
+      # @return [Interface]
+      attr_accessor :interface
+      # @return [String] Bootproto (static, dhcp, none)
```

Comment:
does it make sense for all connection or only physical ones?

Reply:
Why only for physical? BTW, I have not checked all possible scenarios, I started moving the attributes from `LanItems` to the `Interface` and then as we discussed about adding the connection abstraction class .. I moved most of the attributes that seems to be generic to the connection class and the ones specific for some type to the specialized class but some of there probably need to be revised.

Reply:
I didn't see startmode attr. The simplest possible configuration is
```
STARTMODE=auto
BOOTPROTO=dhcp
```

Reply:
BTW: STARTMODE is defined in interface in your opened PR